corehq/apps/export/models/new.py

```diff
@@ -915,6 +924,77 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
```

Comment:
nit: the other instance handle system properties by defining them here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/system_properties.py and then they get inserted in this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/models/new.py#L710.